src/sparseml/optim/helpers.py

```diff
@@ -16,13 +16,17 @@
 Helper functions for base Modifier and Manger utilities
 """
 import json
+import platform
 import re
 import warnings
 from contextlib import suppress
+from copy import deepcopy
 from typing import Any, Dict, Optional, Tuple, Union
 
+import torch
```

Comment:
cannot have a torch specific import in the generic helpers file (ie `keras` and `tensorflow` managers won't work with this), will suggest a small change to `add_framework_metadata` to support adding in extra framework specific metadata